app/services/doc_auth/acuant/pii_from_doc.rb

```diff
@@ -30,15 +30,21 @@ def call
         hash
       end
 
+      ACUANT_TIMESTAMP_FORMAT = %r{/Date\((?<milliseconds>\d+)\)/}.freeze
+
+      # @api private
+      def convert_date(date)
+        match = ACUANT_TIMESTAMP_FORMAT.match(date)
+        return if !match || !match[:milliseconds]
```

Comment:
Just an idea trying to express better the intention.

```suggestion
        return unless Regexp.last_match(:milliseconds)
```

Reply:
TIL about that method 🙂 

I'm not sure I'm a fan in this case because it loses the explicitness of connecting lines 37 and 38 with the variable. It also uses a global variable which can get into weird territory.

Reply:
> it loses the explicitness of connecting lines 37 and 38 with the variable
Yeah, I got it.
I was also wondering if `match[:milliseconds]` would be ever nil/empty

Reply:
Thanks! I agree with @mitchellhenke, going to avoid the global

I would have gone with:
```
match = ACUANT_TIMESTAMP_FORMAT.match(date)
if match && match[:millisecods]
  # ..
end
```
but rubocop got mad at me so we have the guard clause now

Reply:
Other options is the block-way:

```ruby
def convert_date(date)
  ACUANT_TIMESTAMP_FORMAT.match(date) do |match|
    Time.zone.at(match[:milliseconds].to_f / 1000).utc.strftime('%m/%d/%Y')
  end
end
```

No guards, and still concise. Not sure if rubocop would like XD